index.bs

```diff
@@ -863,22 +888,48 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
         imageCapture = new ImageCapture(track);
 
         const capabilities = track.getCapabilities()
-        // Check whether zoom is supported or not.
-        if (!capabilities.zoom) {
-          return;
+        const settings = track.getSettings();
+
+        // Check whether pan is supported or not.
+        if (capabilities.pan) {
+          // Map pan to a slider element.
+          const input = document.querySelector('input[name="pan"][type="range"]');
+          input.min = capabilities.pan.min;
+          input.max = capabilities.pan.max;
+          input.step = capabilities.pan.step;
+          input.value = settings.pan;
+          input.oninput = function(event) {
+            track.applyConstraints({advanced: [{pan: event.target.value}]});
+          };
         }
 
-        // Map zoom to a slider element.
-        const input = document.querySelector('input[type="range"]');
-        input.min = capabilities.zoom.min;
-        input.max = capabilities.zoom.max;
-        input.step = capabilities.zoom.step;
-        input.value = track.getSettings().zoom;
+        // Check whether tilt is supported or not.
+        if (capabilities.tilt) {
+          // Map tilt to a slider element.
+          const input = document.querySelector('input[name="tilt"][type="range"]');
+          input.min = capabilities.tilt.min;
+          input.max = capabilities.tilt.max;
+          input.step = capabilities.tilt.step;
+          input.value = settings.tilt;
+          input.oninput = function(event) {
+            track.applyConstraints({advanced: [{tilt: event.target.value}]});
+          };
+          input.parentElement.hidden = false;
```

Comment:
I think we can remove the `hidden` part as it doesn't add that much value in the context of an example in a spec.

Reply:
Note that it wasn't present in the `pan` example section